browser/net/brave_ads_status_header_network_delegate_helper_unittest.cc

```diff
@@ -239,3 +238,25 @@ TEST_F(AdsStatusHeaderDelegateHelperTest, BraveSearchHostIncognitoProfile) {
 
   EXPECT_FALSE(headers.HasHeader(brave::kAdsStatusHeader));
 }
+
+TEST_F(AdsStatusHeaderDelegateHelperTest,
+       BraveSearchHostIncognitoProfileWhenRewardsEnabledInMainProfile) {
+  profile_->GetPrefs()->SetBoolean(brave_rewards::prefs::kEnabled, true);
+
+  TestingProfile* incognito_profile =
+      TestingProfile::Builder().BuildIncognito(profile_.get());
+
+  auto request_info =
```

Comment:
nit: for consistency do we need _info?

Reply:
yes, because we are creating brave::BraveRequestInfo instance

Reply:
DM'd you. Thanks